src/Toolkit.Forms/TimeSlider/TimeSlider.cs

```diff
@@ -285,9 +285,9 @@ private static void OnTimeStepIntervalPropertyChanged(BindableObject bindable, o
         /// <summary>
         /// Gets the time steps that can be used to set the slider instance's current extent.
         /// </summary>
-        public IReadOnlyList<DateTimeOffset> TimeSteps
+        public IReadOnlyList<DateTimeOffset>? TimeSteps
         {
-            get => NativeSlider.TimeSteps;
+            get => NativeSlider?.TimeSteps;
```

Comment:
NativeSlider cannot be null
```suggestion
            get => NativeSlider.TimeSteps;
```